### ISSUE TITLE
incorporate the swarmkit mode libnetwork scalability test

### ISSUE THREAD

**Issue by huikang** [huikang/libnetwork-scale-test#5]
Hi @dumb0002, my plan is to incorporate your swarmkit emulation [1] to this repo so that we can have a consolidated ansible repo. What do you think?

[1] https://github.com/dumb0002/dumbbr01/tree/master/swarmkit-multi-host-emulator/ansible

**Comment by dumb0002** [comment]
Yes, that sounds good. It would be nice to have a consolidated ansible
repo indeed.

    Thanks,
    Braulio
